pages/index.tsx

```diff
@@ -38,17 +42,20 @@ function Sidebar() {
 
   // NB: hidden on mobile, stats.tsx is shown instead
   return (
-    <div className="pt-28 lg:w-[320px] max-sm:hidden flex flex-col gap-12">
-      {userId && <TrackRecord trackRecordUserId={userId} />}
-      {userId && <Tournaments />}
-      {userId && <UserLists />}
-      {userId && (
-        <QuickFeedback
-          type="Fatebook feedback"
-          placeholder="Give feedback on Fatebook..."
-          style="textarea"
-        />
-      )}
+    <div className="max-sm:hidden flex flex-col gap-12 max-w-[400px]">
+      <div className="min-h-28">{userId && <OnboardingChecklist />}</div>
+      <div className="max-w-[320px] flex flex-col gap-12 ml-auto lg:w-[320px]">
```

Comment:
Changes here:
- Fix width of previous sidebar elements to 320px (mentioned above)
- Allow the onboarding checklist to go up to 400px